lighthouse-core/scripts/lantern/download-traces.sh

```diff
@@ -1,13 +1,21 @@
 #!/bin/bash
 
+set -e
+
 DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 LH_ROOT_PATH="$DIRNAME/../../.."
 cd $LH_ROOT_PATH
 
+if [[ -f lantern-data/lantern-expectations.json ]] && ! [[ "$FORCE" ]]; then
+  echo "Lantern data already detected, done."
```

Comment:
within the `lantern-data/lantern-expectations.json` file....

![image](https://user-images.githubusercontent.com/39191/41436388-597dda78-6fd6-11e8-86e7-49f9c4259961.png)

can these metrics be within an object? like `wptObserved`?